pallets/allocations/src/tests.rs

```diff
@@ -156,12 +156,19 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
 
 	let mut storage = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap();
 
-	pallet_membership::GenesisConfig::<Test> {
+	let rval = pallet_membership::GenesisConfig::<Test> {
 		members: vec![Oracle::get()],
 		..Default::default()
 	}
-	.assimilate_storage(&mut storage)
-	.unwrap();
+	.assimilate_storage(&mut storage);
+
+	if rval.is_err() {
+		log::error!(
```

Comment:
`panic` as you have in the other version is better than `log::error!()` which will only print an error message without chaining the return codes

Reply:
Done